contributors/design-proposals/default-storage-class.md

```diff
@@ -0,0 +1,69 @@
+# Deploying a default StorageClass during installation
+
+## Goal
+
+Usual Kubernetes installation tools should deploy a default StorageClass
+where it makes sense.
+
+"*Usual installation tools*" are:
+
+* cluster/kube-up.sh
+* kops
+* kubeadm
+
+Other "installation tools" can (and should) deploy default StorageClass
+following easy steps described in this document, however we won't touch them
+during implementation of this proposal.
+
+"*Where it makes sense*" are:
+
+* AWS
+* Azure
+* GCE
+* Photon
+* OpenStack
+* vSphere
+
+Explicitly, there is no default storage class on bare metal.
```

Comment:
I think it should specify that there is no **provided** default storage class for bare metal.. e.g. things like [nfs-provisioner](https://github.com/kubernetes-incubator/nfs-provisioner) exists and someone can write their own.